webapi/src/main/scala/org/knora/webapi/responders/v1/ValuesResponderV1.scala

```diff
@@ -345,6 +349,130 @@ class ValuesResponderV1 extends ResponderV1 {
     }
 
     /**
+      * Temporary structure to represent existing file values of a resource.
+      *
+      * @param property       the property Iri (e.g., hasStillImageFileValueRepresentation)
+      * @param valueObjectIri the Iri of the value object.
+      * @param quality        the quality of the file value
+      */
+    private case class CurrentFileValue(property: IRI, valueObjectIri: IRI, quality: Option[Int])
+
+    private def changeFileValueV1(changeFileValueRequest: ChangeFileValueRequestV1): Future[ChangeFileValueResponseV1] = {
```

Comment:
Actually I thought that Sipi would just create a new file that would kept track of by Knora (the already existing files would still be referenced by previous versions of the file value). 

Does Sipi actually has to know that files are related to one another as versions? I just stores and serves files. Also, a new version of a file value could be totally different binary.